src/generator/generator_visitor.ts

```diff
@@ -413,6 +420,25 @@ class GeneratorVisitor extends Visitor {
         return 'label_' + this.labelCounter++;
     }
 
+    private getOperationType(expression: BabelExpression) {
+        const type = this.getExpressionType(expression);
+
+        if (type === WebAssemblyType.FLOAT_64) {
```

Comment:
Move this part to the `WebAssemblyType` enum declaration.

Reply:
This doesn't work, because it accesses a property of the class.